### ISSUE TITLE
 Favicon won't update

### ISSUE THREAD

**Issue by Abhi-Raj** [photobox/pbx-group-security#66]

**Comment by fzipi** [comment]
There are many sizes of favicon required in this theme:
```
<link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png">
<link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png">
<link rel="apple-touch-icon" sizes="72x72" href="/apple-icon-72x72.png">
<link rel="apple-touch-icon" sizes="76x76" href="/apple-icon-76x76.png">
<link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png">
<link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png">
<link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png">
<link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
<link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
<link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
```
And there is one `favicon.ico` in the `static/img` folder which isn't linked anywhere. 

So, are there any other favicon images of some quality that we can use? Vectorial maybe?

With that it can easily generate favicon for a variety of display sizes.

If you just want a favicon, we can use the one provided and just comment out all those.

**Comment by Abhi-Raj** [comment]
@fzipi - Let me check the favicons with designer and come back

**Comment by Abhi-Raj** [comment]
@fzipi We can use this consistently (https://github.com/photobox/pbx-group-security/blob/master/static/img/favicon.ico). If we need a higher resolution of this icon, can ask our designer to share.

**Comment by fzipi** [comment]
Fixed favicon in commit 3781d57.
